cpp/include/cudf/detail/aggregation/aggregation.cuh

```diff
@@ -129,6 +129,34 @@ struct update_target_element<
   }
 };
 
+template <typename Source, bool target_has_nulls, bool source_has_nulls>
+struct update_target_element<Source,
+                             aggregation::MIN,
+                             target_has_nulls,
+                             source_has_nulls,
+                             std::enable_if_t<is_fixed_point<Source>()>> {
+  __device__ void operator()(mutable_column_device_view target,
+                             size_type target_index,
+                             column_device_view source,
+                             size_type source_index) const noexcept
+  {
+#if (__CUDACC_VER_MAJOR__ != 10) or (__CUDACC_VER_MINOR__ != 2)
+
+    if (source_has_nulls and source.is_null(source_index)) { return; }
+
+    using Target       = target_type_t<Source, aggregation::MIN>;
+    using DeviceTarget = device_storage_type_t<Target>;
+    using DeviceSource = device_storage_type_t<Source>;
```

Comment:
Instead of adding a new partial specialization, can't the `device_storage_type_t` logic just be added to the existing specialization for `is_fixed_width`?

Reply:
That is one of the MANY things I tried, but it causes the NVCC to fall over and fail (for even more than just the hash implementation). This was the only thin that I could find that worked. When we upgrade to CUDA 11, I will replace everything with what you are suggesting.

Reply:
Got it, makes sense.